src/cache/DefaultCache.php

```diff
@@ -16,7 +19,7 @@ class DefaultCache implements CacheInterface
 
     public function __construct(int $maxsize, int $ttl)
     {
-        $this->cache = new \Sabre\Cache\Memory();
+        $this->cache = new FilesystemAdapter();
```

Comment:
Is this still an in-memory cache or is it now on the filesystem?

Reply:
It is on filesystem.

Reply:
It shouldn't be

Reply:
Updated to `ArrayAdapter` which is an in-memory cache